Source/WebCore/platform/network/NetworkStorageSession.h

```diff
@@ -386,14 +389,10 @@ class NetworkStorageSession
     HashSet<RegistrableDomain> m_managedDomains;
 
 #if PLATFORM(COCOA)
-public:
-    CookieStorageObserver& cookieStorageObserver() const;
-
-private:
     mutable std::unique_ptr<CookieStorageObserver> m_cookieStorageObserver;
 #endif
     uint64_t m_cookiesVersion { 0 };
-    HashMap<uint64_t, Vector<CompletionHandler<void()>>> m_cookiesVersionChangeCallbacks;
+    Deque<std::pair<uint64_t, CompletionHandler<void()>>> m_cookiesVersionChangeCallbacks;
```

Comment:
Instead of a pair, let's use a struct, to make it clear that the first data member is a version.